db/reset_db.js

```diff
@@ -1,7 +1,19 @@
 const { User, FrequentProblem } = require('../models')
 
 User.sync({force: true})
-FrequentProblem.sync({force: true})
+FrequentProblem.sync({force: true}).then(function () {
+
+    FrequentProblem.destroy({truncate: true})
```

Comment:
update - something is weird for the reset_db with postgres.. logged it on trello